### DISCUSSION TITLE
App Directory Tauri Alias / Open App from Tauri

### DISCUSSION THREAD

**Discussion by MentalGear** [tauri-apps/tauri#4507] [Q&A]
I checked the doc, but I didn't find a Tauri alias for the System's app/program directory. For example on MacOS (/Applicatons and /Users/[username]/Applications).

On a related note:
Is it possible to start an app via Tauri? For example, if I want open MS Word via Tauri?

**Comment by wkl007** [comment]
I have the same problem

**Comment by FabianLars** [comment]
For the first part i'm going to copy my [discord messages](https://discord.com/channels/616186924390023171/1007685382801989753/1007690128870867044) as a start:
> So on macos i think it's always just `/Applications`
> On Linux there are too many possibilities tbh, there is no real standard here afaik
> On Windows it's tricky, like there's an api for it, but it's kinda hard to use from rust, the best crate i could find rn is this one https://crates.io/crates/windirs
> Here's a small discussion on why it's not part of most of the dirs crates https://github.com/dirs-dev/dirs-rs/pull/10

tldr: It's unlikely that we will add this function to our api.

> Is it possible to start an app via Tauri? For example, if I want open MS Word via Tauri?

 if it's registered as the standard app for a file you wanna open you can use https://tauri.app/v1/api/js/modules/shell#open, otherwise you need to find the executable on the system (good luck if it's a MS Store app :/ ) and spawn that with a (detached) command or something 🤔
